README.md

```diff
@@ -13,7 +13,9 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 
 * Service APIs for “Quality on Demand” (see APIBacklog.md)
 * It provides the customer with the ability to:  
-  * set quality for access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)  
+  * set quality for access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)
+      * Session mode, for a specific duration
```

Comment:
```suggestion
      * Session mode, for a specific duration for the device
```

Adding "the device" to be consistent with the second bullet.

Reply:
Actually is the quality set for the specified connection, not for the complete device, therefore this addition could be misleading.
My understanding is that in the second bullet "device connects to the network" is the trigger to set the quality for the (specific) connection again, but it is not set for the complete device.